manager/integration/tests/test_csi_snapshotter.py

```diff
@@ -32,7 +32,7 @@ class VolumeSnapshotClassFactory():
         manifests = []
 
         @staticmethod
-        def create_volumesnapshotclass(name, deletepolicy):
+        def create_volumesnapshotclass(name, deletepolicy, param=None):
```

Comment:
Suggest using an explanatory parameter instead of a general one. ex: `snapshot_type` instead of `param`.

You should change other places as well.

Reply:
Hi @innobead ,
I changed parameter name for you review, thanks